tower-buffer/src/layer.rs

```diff
@@ -12,6 +12,7 @@ pub struct BufferLayer<Request, E = DefaultExecutor> {
 }
 
 impl<Request> BufferLayer<Request, DefaultExecutor> {
+    #[allow(missing_docs)]
```

Comment:
Why are we allowing this?

Reply:
Because having docs on every `new` method that says "Creates a new Foo" seemed unnecessary, and I don't want to _also_ write expansive docs for all the methods in this patch :)

Reply:
works for me 😄